tests/setuponlytests/vanillatweaks_file/verify.sh

```diff
@@ -0,0 +1,4 @@
+mc-image-helper assert fileExists /data/world/datapacks/afk*
+mc-image-helper assert fileExists /data/world/datapacks/graves*
```

Comment:
I think will need to be quoted to avoid the shell expanding the wildcard instead of `assert fileExists`. Like
```suggestion
mc-image-helper assert fileExists "/data/world/datapacks/afk*"
mc-image-helper assert fileExists "/data/world/datapacks/graves*"
```